test/config.js

```diff
@@ -0,0 +1,23 @@
+const config = {
+  database: {
+    datatype: "mysql",
+    host: "localhost",
+    name: "test",
+    user: "root",
+    password: "",
+    charset: "utf8mb4",
+    version: "2",
+  },
+  api: {
+    endpoint: "/api",
+    version: "1",
+    port: 8081,
+  },
+  auth: {
+    endpoint: "/auth",
+    version: "1",
+    port: 8085,
+  },
+};
+
+export default () => config;
```

Comment:
You dont need to do that. export the `config` object on line 1 insteaad:

```
export const config = {
};
```

Reply:
For this it's requested by EsLint : 
```[eslint] Prefer default export. (import/prefer-default-export)```

Reply:
You can `export default { your config object here };` or disable this rule for this file.

Reply:
I try that